.github/workflows/dev-release-marimo-base.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish marimo-base dev release
+
+# Publish development release to test pypi on pushes to main
+on:
+  push:
+    branches:
+      - main
+
+env:
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  TURBO_TEAM: marimo
+
+jobs:
+  publish_dev_release:
+    name: 📤 Publish dev release for marimo-base
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.12.1
+
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v4
+        with:
+          # get tag history for version number
+          fetch-depth: 0
+
+      - name: 🥚 Install Hatch
+        uses: pypa/hatch@install
+
+      # Python build tools generally only allow building one project
+      # per Python project. This modifies the pyproject.toml
+      # to build marimo-base, a slimmed down marimo distribution with
+      # no static artifacts.
+      # 
+      # Adapted from https://github.com/cvxpy/cvxpy/blob/297278e2a88db3c0084750052a16e60672074da3/.github/workflows/build.yml#L169C1-L180C1
+      - name: Adapt pyproject.toml to build marimo-base
+        run: |
+          # Mac has a different syntax for sed -i, this works across oses
+          sed -i.bak -e 's/name = "marimo"/name = "marimo-base"/g' pyproject.toml
+          # Remove static artifacts
+          sed -i.bak '/artifacts = /d' pyproject.toml
```

Comment:
we could put this in `pyproject.toml` under a different target name: https://hatch.pypa.io/1.9/config/build/#pyprojecttoml_18

`wheel-slim`

Reply:
It's still going to be released as `marimo` on the package indices though. Whereas want to create a separate package called `marimo-base`.

Reply:
Oh I see what you're saying. Additionally I couldn't figure out how to get a custom target to build a wheel.

Reply:
Oh maybe hatch doesn't allow you to specify a target